src/bundle/Resources/config/services.yml

```diff
@@ -68,3 +68,11 @@ services:
         public: false
 
     EzSystems\HybridPlatformUi\Filter\VersionFilter: ~
+
+    EzSystems\HybridPlatformUi\Repository\PathService:
+        arguments:
+            - "@ezpublish.api.service.inner_search"
```

Comment:
Why did you use `@ezpublish.api.service.inner_search` instead of `@ezpublish.api.service.search` ?